src/rgw/rgw_iam_policy.h

```diff
@@ -359,6 +360,18 @@ struct Condition {
     }
   };
 
+  struct ci_string_like : public std::binary_function<const std::string,
+                                                     const std::string,
+                                                     bool> {
+    bool operator ()(const std::string& s1,
+                    const std::string& s2) const {
+      if(fnmatch(s2.c_str(), s1.c_str(), 0) == 0){
```

Comment:
it doesn't look like [fnmatch()](http://man7.org/linux/man-pages/man3/fnmatch.3.html) defaults to case-insensitive matching:
> FNM_CASEFOLD: If this flag (a GNU extension) is set, the pattern is matched case-insensitively.